server/fleetv2_http_api/models/message.py

```
@@ -2,13 +2,13 @@
-from server.fleetv2_http_api.models.base_model import Model
-from server.fleetv2_http_api.models.device_id import DeviceId
-from server.fleetv2_http_api.models.payload import Payload
-from server.fleetv2_http_api import util
+from fleetv2_http_api.models.base_model import Model
+from fleetv2_http_api.models.device_id import DeviceId
+from fleetv2_http_api.models.payload import Payload
+from fleetv2_http_api import util
 
-from server.fleetv2_http_api.models.device_id import DeviceId  # noqa: E501
-from server.fleetv2_http_api.models.payload import Payload  # noqa: E501
+from fleetv2_http_api.models.device_id import DeviceId  # noqa: E501
```

Ruff (0.8.2)

Redefinition of unused `DeviceId` from line 6

Remove definition: `DeviceId`

(F811)